study-builder/studies/circadia/study.conf

```diff
@@ -654,6 +654,102 @@ user.studies["circadia"].forms["PREQUAL"].questions["VISUAL_PROBLEM_1"].answers.
       "maxOccurrencesPerUser": 1,
       "dispatchToHousekeeping": true,
       "order": 1
-    }
+    },
+    // Complete consent reminder 2 days
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          {
+            "emailTemplate": ${emails.complete_activity_reminder_en},
+            "language": "en",
+            "isDynamic": true
+          }
+        ],
+        "linkedActivityCode": "CONSENT",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["circadia"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 172800, // 2 days in seconds
```

Comment:
One thing I like to do is pull these out into constants in the substitutions file (e.g. `subs.conf`). That way, it is much easier to tweak these timeouts in one place. So in local/dev environment, we can set value to minutes rather than days.